src/main/java/seedu/spendnsplit/storage/JsonUserPrefsStorage.java

```diff
@@ -55,8 +56,8 @@ public Optional<UserPrefs> readUserPrefs(Path prefsFilePath) throws DataLoadingE
         if (!readValue.isPresent()) {
             return readValue;
         }
-        if (readValue.orElse(new UserPrefs()).getAddressBookFilePath() == null) {
-            readValue.orElse(new UserPrefs()).setAddressBookFilePath(Paths.get("data" , "addressbook.json"));
+        if (readValue.orElse(new UserPrefs()).getSpendNSplitBookFilePath() == null) {
+            readValue.orElse(new UserPrefs()).setSpendNSplitBookFilePath(Paths.get("data" , "addressbook.json"));
```

Comment:
The json file should be changed

Reply:
Yup, caught it and another area where I didnt change